tests/functional/object/mcg/test_block_external_access.py

```diff
@@ -156,12 +158,43 @@ def test_block_access_from_storagecluster(
 
         self.check_disable_routes(False)
 
+        bucket_name = bucket_factory(
+            1,
+            interface="CLI",
+            bucketclass={
+                "interface": "CLI",
+                "backingstore_dict": {"aws": [(1, "eu-central-1")]},
+            },
+        )[0].name
+
+        standard_test_obj_list = awscli_pod_session.exec_cmd_on_pod(
+            f"ls -A1 {AWSCLI_TEST_OBJ_DIR}"
+        ).split(" ")
+        file_name = standard_test_obj_list[0]
+        logger.info(f"Going to copy file {file_name} to the bucket {bucket_name}")
+        awscli_pod_session.exec_cmd_on_pod(
+            command=craft_s3_command(
+                f"cp {AWSCLI_TEST_OBJ_DIR}{file_name} s3://{bucket_name}/{file_name}",
+                mcg_obj=mcg_obj,
+            ),
+            out_yaml_format=False,
+        )
+
         self.set_disable_routes_value(storagecluster_obj, True, True)
         self.check_disable_routes(True)
 
         self.set_disable_routes_value(storagecluster_obj, True, False)
         self.check_disable_routes(False)
 
+        logger.info(f"Going to copy file {file_name} to the bucket {bucket_name}")
+        awscli_pod_session.exec_cmd_on_pod(
+            command=craft_s3_command(
+                f"cp {AWSCLI_TEST_OBJ_DIR}{file_name} s3://{bucket_name}/{file_name}",
+                mcg_obj=mcg_obj,
+            ),
+            out_yaml_format=False,
+        )
```

Comment:
could we add assert statement to check if the expected results are acheived.
i.e. 
- if disableRoutes flag in storagecluster yaml is set to False then all routes are recreated after deletion
- if disableRoutes flag in storagecluster yaml is set to True then noobaa-mgmt, s3 and sts routes are not
        recreated after deletion